### DISCUSSION TITLE
Can no longer dl from site soap2day.rs

### DISCUSSION THREAD

**Discussion by itrob2023** [aclap-dev/video-downloadhelper#136] [Help and Q&A]
### Your question

Greetings- 
Look, I've been reading the comments and seems like you're pretty busy with fixing the latest update so I'm not here to bash you. You've been asking for the URL to the download sites so here is mine: soap2day.rs and I'm running Windows.
Like others I've read, the CoApp no longer copies the URL. Hope you can fix, and thanks for what you do.

**Comment by paulrouget** [comment]
Thank you! I'll look into it.

**Comment by paulrouget** [comment]
> soap2day.rs

Seems to work for me.

Do you mind sharing more information about your setup: https://github.com/aclap-dev/video-downloadhelper/wiki/Where-can-I-find-my-versions%3F

>  the CoApp no longer copies the URL

Upcoming release will fix that.

**Comment by paulrouget** [comment]
> Can no longer dl

Can you share more details of how this fails? You get an error?

**Comment by itrob2023** [comment]
I dl by copying the URL first from the CoApp, then paste into Video Downloadhelper. Since I can't copy the URL, I can no longer download.

Downloading straight from VD always gives me issues. The sound is unsynched by the end of the video if over an hour. So I started copying the URL and that's been working fine for me.

Sent with [Proton Mail](https://proton.me/) secure email.

**Reply by paulrouget** [reply]
The copy thing will be fixed in the upcoming release.

**Comment by Mattmjs** [comment]
>  The sound is unsynched by the end of the video if over an hour

That should not happen on the latest companion app versions that come with ffmpeg. It did that on older 1x app versions but you don't indicate which version you use. You've provided no technical information.
